reference/v8.json

```diff
@@ -1861,6 +1904,9 @@
       "sdk-support": {
         "basic functionality": {
           "js": "0.27.0"
+        },
+        "data-driven styling": {
+          "js": "Not supported"
```

Comment:
Settling for a prose solution:

![image](https://cloud.githubusercontent.com/assets/98601/22356365/9345df5e-e3e5-11e6-8176-2f2764f2f7b9.png)